app/assets/appfilter.xml

```diff
@@ -3100,6 +3104,7 @@
   <item component="ComponentInfo{cc.forestapp/cc.forestapp.activities.PlantView.PlantViewController}" drawable="forest" name="Forest" />
   <item component="ComponentInfo{cc.forestapp/cc.forestapp.activities.launch.LaunchActivity}" drawable="forest" name="Forest" />
   <item component="ComponentInfo{cc.forestapp/cc.forestapp.activities.main.MainActivity}" drawable="forest" name="Forest" />
+  <item component="ComponentInfo{kaka.wallpaper.forest/kaka.wallpaper.forest.android.SettingsActivity}" drawable="forest_settings" name="Forest Settings" />
```

Comment:
Isn't this **Forest Live Wallpaper**?

Reply:
![Screenshot_20240505-055533.jpg](https://github.com/LawnchairLauncher/lawnicons/assets/122789671/bd564766-eea5-411f-8e3e-d2488abbce5c)

Reply:
Somehow the app name is different than the playstore name. Which one should I use?

Reply:
> Which one should I use?

Forest Live Wallpaper. I couldn't find this app by the name "Forest Settings", so I think others are also most likely searching by the name from Google Play.

Reply:
oh ok gimme sec

Reply:
drawable="forest_settings" → drawable="forest_live_wallpaper"